shared/product.go

```diff
@@ -111,3 +112,26 @@ func CertRotate(product string, ips []string) (string, error) {
 
 	return "", nil
 }
+
+func SecretEncryptOps(action, ip, product string) (string, error) {
+	secretEncryptCmd := map[string]string{
+		"status":      fmt.Sprintf("sudo %s secrets-encrypt status", product),
+		"enable":      fmt.Sprintf("sudo %s secrets-encrypt enable", product),
+		"disable":     fmt.Sprintf("sudo %s secrets-encrypt disable", product),
+		"prepare":     fmt.Sprintf("sudo %s secrets-encrypt prepare", product),
+		"rotate":      fmt.Sprintf("sudo %s secrets-encrypt rotate", product),
+		"reencrypt":   fmt.Sprintf("sudo %s secrets-encrypt reencrypt", product),
+		"rotate-keys": fmt.Sprintf("sudo %s secrets-encrypt rotate-keys", product),
+	}
+
+	stdout, err := RunCommandOnNode(secretEncryptCmd[action], ip)
```

Comment:
is there any other more meaningful name to this var instead of stdout ? we could be more specific here

Reply:
secretsEncryptStdOut ?

Reply:
yeah anything that could relate to the output

Reply:
fixed